src/lotto/LottoResultPrinter.js

```diff
@@ -0,0 +1,47 @@
+import { MESSAGES } from '../constants.js';
+import { print } from '../utils/index.js';
+
+class LottoResultPrinter {
+  #matchedCountPerMatchOption;
+
+  #price;
+
+  constructor(matchedCountPerMatchOption, price) {
+    this.#matchedCountPerMatchOption = matchedCountPerMatchOption;
+    this.#price = price;
+  }
+
+  printResult() {
+    print(MESSAGES.IO.OUTPUT.WINNING_STATISTICS);
+    this.#printWinningResult();
+    this.#printRateOfReturn();
+  }
+
+  #printWinningResult() {
+    const matchResults = this.#matchedCountPerMatchOption.map(this.#formatMatchResult);
+    print(matchResults.join('\n'));
+  }
+
+  #formatMatchResult({ count, matchedCount, isBonus, prize }) {
+    if (isBonus) {
+      return MESSAGES.IO.OUTPUT.BONUS_MATCH_RESULT(count, prize, matchedCount);
+    }
+
+    return MESSAGES.IO.OUTPUT.MATCH_RESULT(count, prize, matchedCount);
+  }
+
+  #printRateOfReturn() {
+    const totalPrize = this.#matchedCountPerMatchOption.reduce((acc, cur) => {
+      const { matchedCount, prize } = cur;
+
+      return acc + matchedCount * prize;
+    }, 0);
+
+    const rateOfReturnPercent = (totalPrize / this.#price) * 100;
+    const rateOfReturn = rateOfReturnPercent.toFixed(1);
```

Comment:
```
- 수익률은 소수점 둘째 자리에서 반올림합니다.
   - ex. 100.0%, 51.5%, 1,000,000.0%
```
이런 항목이 있었죠. 이런 상황에서는 소수점만 찍는 toFixed보다 천 단위마다 콤마를 찍어주는 toLocaleString이 적절하다는 피드백을 봤었어요! 만약 적용하면 이런 식이 될 것 같아요!~
```suggestion
    const rateOfReturnPercent = (totalPrize / this.#price) * 100;
    const rateOfReturn = rateOfReturnPercent.toLocaleString('ko-KR', {
      minimumFractionDigits: 1,
      maximumFractionDigits: 1,
    });
```

Reply:
오, 천 단위 콤마는 생각 못했는데, toLocaleString이 천 단위 콤마도 찍어주면서 소수점도 몇 자리까지 표시할 건지 지정해줄 수 있군요! 좋은 방법 알려주어 감사합니다:)